klib/src/platform/windows/GDIPlusFlat.def

```diff
@@ -0,0 +1,5 @@
+package = platform.GDIPlusFlat
```

Comment:
You likely needs depends=windows

Reply:
Not sure, but OK - it's windows package anyway :)

Reply:
Hm, strange.
In generated ```GDIPlusFlat.kt``` there are bunch of inports like
```
import platform.posix.BITMAPINFO
import platform.posix.BOOL
import platform.posix.BOOLVar
import platform.posix.BYTEVar
import platform.posix.CLSID
...
```
Why ```posix```, shouldn't be ```windows```?

Reply:
Fixed - now MinGW platform.posix depends on platform.windows
And it really is just emulation of POSIX API over Windows.